docs/content/troubleshooting/troubleshoot-ingress-controller.md

```diff
@@ -47,102 +47,144 @@ To check the Ingress Controller logs -- both of the Ingress Controller software
-
-### Checking the Events of a VirtualServer and VirtualServerRoute Resources
-
-After you create or update a VirtualServer resource, you can immediately check if the NGINX configuration for that  resource was successfully applied by NGINX:
-```
-$ kubectl describe vs cafe
-. . .
-Events:
-  Type    Reason          Age   From                      Message
-  ----    ------          ----  ----                      -------
-  Normal  AddedOrUpdated  16s   nginx-ingress-controller  Configuration for default/cafe was added or updated
-```
-Note that in the events section, we have a `Normal` event with the `AddedOrUpdated` reason, which informs us that the configuration was successfully applied.
-
-Checking the events of a VirtualServerRoute is similar:
-```
-$ kubectl describe vsr coffee
-. . .
-Events:
-  Type     Reason                 Age   From                      Message
-  ----     ------                 ----  ----                      -------
-  Normal   AddedOrUpdated         1m    nginx-ingress-controller  Configuration for default/coffee was added or updated
-```
-
-### Checking the Events of a Policy Resource
-
-After you create or update a Policy resource, you can use `kubectl describe` to check whether or not the Ingress Controller accepted the Policy:
-```
-$ kubectl describe pol webapp-policy
-. . .
-Events:
-  Type    Reason          Age   From                      Message
-  ----    ------          ----  ----                      -------
-  Normal  AddedOrUpdated  11s   nginx-ingress-controller  Policy default/webapp-policy was added or updated
-```
-Note that in the events section, we have a `Normal` event with the `AddedOrUpdated` reason, which informs us that the policy was successfully accepted.
+There are two settings that need to be set to enable more debug/verbose logging for NGINX Ingress controller.
 
-However, the fact that a policy was accepted doesn't guarantee that the NGINX configuration was successfully applied. To confirm that, check the events of the VirtualServer and VirtualServerRoute resources that reference that policy.
+1. command line arguments
+2. configmap settings
+
+This document will cover how to enable both.
+
+
+You can use NGINX Ingress controller command line arguments. This is a great way to increase debug log levels for both Ingress as well as NGINX.
+When using `manifest` for deployment:
+
+1.) Use the command line argument, `- -nginx-debug` in your deployment/daemonset
+2.) If you want to increase the Ingress `Controller` process lets, set the command line argument to `v=3`, which collects more information on Ingress
+
+ ```yaml
+ -v=3
+ ```
+
+Here is a small snippet of setting these a command line arguments in the `ARGS` section of a deployment:
+
+```yaml
+args:
+  - -nginx-configmaps=$(POD_NAMESPACE)/nginx-config
+  - -enable-cert-manager
+  - -nginx-debug
+  - -v=3
+```
+
+You can configure `error-log-level` in the NGINX Ingress controller `configMap`:
```

Comment:
```suggestion
You can configure `error-log-level` in the Ingress Controller `configMap`:
```

Reply:
```suggestion
You can configure `error-log-level` in the NGINX Ingress Controller `configMap`:
```

The name should be capitalised, but we actually keep the 'NGINX' prefix to product names at all times.

Reply:
this is not consistent in the docs and we actually have more without `NGINX`...do we have to update them all to add it?